site/_docs/history.md

```diff
@@ -56,19 +146,297 @@ other software versions as specified in gradle.properties.
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6220">CALCITE-6220</a>]
+ Rewrite `MIN`/`MAX`(bool) as `BOOL_AND`/`BOOL_OR` for Postgres, Redshift
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6321">CALCITE-6321</a>]
+ Add `copy(List<RexLiteral>)` method to Window class
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6337">CALCITE-6337</a>]
+ Distinguish naked measure support between inside and outside aggregation
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6323">CALCITE-6323</a>]
+ Serialize return type during `RelJson.toJson(RexNode node)` for `SqlKind.SAFE_CAST`
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6111">CALCITE-6111</a>]
+ Explicit cast from expression to numeric type doesn't check overflow
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6162">CALCITE-6162</a>]
+ Add rule(s) to remove joins with constant single tuple relations
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6192">CALCITE-6192</a>]
+ `DEFAULT` expression with `NULL` value throws unexpected exception
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6147">CALCITE-6147</a>]
+ `CAST(CAST(EMPNO AS VARCHAR) AS INT)` should be simplified to `EMPNO`
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6102">CALCITE-6102</a>]
+ `SqlWriter` in `SqlInsert`'s unparse start a list but does not end it
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5607">CALCITE-5607</a>]
+ Serialize return type during `RelJson.toJson(RexNode node)` for `SqlKind.MINUS`
+Revert * [<a href="https://issues.apache.org/jira/browse/CALCITE-6222">CALCITE-6222</a>]
+ Mysql does not have `to_char` function
+Revert * [<a href="https://issues.apache.org/jira/browse/CALCITE-6180">CALCITE-6180</a>]
+ Append possibility to escape backslash in `LIKE` with `ESCAPE` operator
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6269">CALCITE-6269</a>]
+  Fix missing/broken BigQuery date-time format elements
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6231">CALCITE-6231</a>]
+  JDBC adapter generates `UNNEST` when it should generate `UNNEST ... WITH ORDINALITY`
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6208">CALCITE-6208</a>]
+  Update `JSON_VALUE` return type inference to make explicit array return types be nullable with nullable elements
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6199">CALCITE-6199</a>]
+  Trim unused fields for `SNAPSHOT` and `SAMPLE` if table has `VIRTUAL` column
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6063">CALCITE-6063</a>]
+  If `ARRAY` subquery has `ORDER BY` (without `LIMIT`), rows are not sorted
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6032">CALCITE-6032</a>]
+  Multilevel correlated query is failing in `RelDecorrelator` code path
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6254">CALCITE-6254</a>]
+  Support table function calls in `FROM` clause without `TABLE()` wrapper
```

Comment:
This one also seems like a "feature". And even more, I think it's worth to highlight this feature, what do you think?